coffee_core/src/coffee.rs

```diff
@@ -227,17 +228,42 @@ impl PluginManager for CoffeeManager {
         for repo in self.repos.values() {
             if let Some(mut plugin) = repo.get_plugin_by_name(plugin) {
                 log::trace!("{:#?}", plugin);
-                let result = plugin.configure(verbose).await;
-                log::debug!("result from plugin configure: {:?}", result);
-                match result {
-                    Ok(path) => {
+
+                // old_root_path is the path where the plugin is cloned and currently stored
+                // eg. ~/.coffee/repositories/<repo_name>/<plugin_name>
+                let old_root_path = plugin.root_path.clone();
+                // new_root_path is the path where the plugin will be installed specific to the network
+                // eg. ~/.coffee/<network>/plugins/<plugin_name>
+                let new_root_path = format!(
+                    "{}/{}/plugins/{}",
+                    self.config.root_path,
+                    self.config.network,
+                    plugin.name()
+                );
+                dir::copy(
+                    &old_root_path,
+                    format!("{}/{}/plugins/", self.config.root_path, self.config.network),
+                    &dir::CopyOptions::new(),
+                )
+                .map_err(|err| error!("{}", err.to_string()))?;
```

Comment:
Please see the fix https://github.com/coffee-tools/coffee/pull/180/commits/7edbc1bbb462a4a9f78e61726616fb192eedce19 this also make the copy async and more easy to read

This should work also for the double copy of the same dir two time :) Easy

Reply:
Nice. Thanks!